src/pc/log.c

```diff
@@ -4,7 +4,7 @@
 #include <string.h>
 
 // uncomment to disable the logs grouping feature
-// #define FORCE_LINEAR_LOGS
+#define FORCE_LINEAR_LOGS
```

Comment:
Let's comment this back?

Reply:
Done, this is really slow on my system though. I think we should reduce the amount of logging